tests/plugins/fakeobjects/manual/pastefakeelementwithimage.md

```diff
@@ -0,0 +1,17 @@
+@bender-tags: 4.8.0, bug, clipboard, 638
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, basicstyles, iframe, clipboard, sourcearea, link, image2
+
+**Scenario:**
+
+1. Open this page in new tab.
+2. Click `Source` in editor and paste:
```

Comment:
This step could be omitted and instead this HTML could be already inserted into the editor.